code/datums/uplink_item.dm

```diff
@@ -991,6 +991,7 @@ var/list/uplink_items = list()
 	reference = "BRHS"
 	item = /obj/item/weapon/storage/box/syndie_kit/hardsuit
 	cost = 8
+	excludefrom = list(/datum/game_mode/nuclear)
```

Comment:
They can buy syndicate space suits and the elite hardsuits, but not their own? Seems a bit weird.